doc/protocol.md

```diff
@@ -778,8 +774,6 @@ payload type:
 
 You may also specify these options:
 
- * "connection": A stable identifier for connection sharing, i.e.
-   sending multiple requests to a single open connection.
```

Comment:
The docbook docs still mention "connection" as an option (doc/guide/cockpit-http.xml)

And iirc changing that is fine but just needs a mention in the conversion PR https://github.com/cockpit-project/cockpit/pull/21515

Reply:
Right, dropped from the guide, thanks!